src/lib/footer/Footer.svelte

```diff
@@ -0,0 +1,161 @@
+            <li class="footer__algemeen-links"><a href="/">Pers</a></li>
+        </ul>
+        <span class="footer__underline"></span>
+        <p class="footer__copyright">&copy; Mediahuis 2024</p>
+        <ul class="footer__FAQ">
+            <li class="footer__FAQ-links"><a href="/">Privacy verklaring</a></li>
+            <li class="footer__FAQ-links"><a href="/">Gebruiksvoorwaarden</a></li>
+            <li class="footer__FAQ-links"><a href="/">Cookieverklaring</a></li>
+            <li class="footer__FAQ-links"><a href="/">Cookie voorkeuren</a></li>
+        </ul>
+    </div>
+
+</footer>
+
+<style>
+    .footer__h3 {
+        font-style: var(--h3-text-size);
+    }
+
+    .footer {
+        background-color: var(--color-footer);
+        padding: 3.5rem;
+    }
+
+    .footer__grid {
+        font-family: var(--font-family);
+        color: var(--light);
+        display: grid;
+        grid-template-columns: repeat(4, 0.3fr);
+        grid-template-rows: repeat(5, 0.3fr);
+        padding: 3.5rem;
+        grid-template-columns: repeat(4, 0.5fr);
+        grid-template-rows: repeat(5, 1fr);
+        gap: 1rem;
+        grid-template-areas:
+        "veronica . . ."
+        "header-footer . radio-veronica algemeen"
+        "socials . radio-veronica algemeen"
+        "space space space space"
+        "copyright . FAQ FAQ"
+        ;
+    }
+
+    .footer__veronica {
+        grid-area: veronica;
+        width: 6rem;
+        height: 2.5rem;
+    }
+
+    .footer__h2 {
+        grid-area: header-footer;
+        width: 13rem;
+        font-size: 2.2rem;
+    }
+
+    .footer__radio-veronica {
+        grid-area: radio-veronica;
+        display: flex;
+        flex-direction: column;
+        gap: 1rem;
+    }
+
+    .footer__algemeen {
+        grid-area: algemeen;
+        display: flex;
+        flex-direction: column;
+        gap: 1rem;
+    }
+
+    .footer__underline {
+        grid-area: space;
+        width: 100%;
+        height: 0.1rem;
+        background-color: var(--secondary-color);
+    }
+
+    .footer__socials {
+        grid-area: socials;
+        display: flex;
+        flex-direction: column;
+        gap: 1rem;
+    }
+
+    .footer__socials-icons {
+        display: flex;
+        flex-direction: row;
+        gap: 1rem;
+    }
+
+    .footer__icons {
+        background-color: var(--secondary-color);
+        display: flex;
+        border-radius: 3rem;
+        width: 2rem;
+        height: 2rem;
+        justify-content: center;
+        padding: 0.5rem;
+        cursor: pointer;
+    }
+
+    .footer__icons:hover {
+        background-color: var(--tertairy);
+    }
+
+    .footer__icons img {
+        width: 1rem;
+        height: 1rem;
+    }
+
+    .footer__copyright {
+        grid-area: copyright;
+        font-size: var(--p-text-size);
+    }
+
+    .footer__FAQ {
+        grid-area: FAQ;
+        display: flex;
+        flex-direction: row;
+        gap: 1rem;
+    }
+
+    li {
```

Comment:
Ik zou je li element bovenaan je css zetten omdat elementen hoger staan dan classes, zo blijft het overzichtelijker en kun je er makkelijker achterkomen als een class zichzelf overchrijft.

Reply:
het is opgelost

Reply:
Lekker bezig!